main.tf

```diff
@@ -93,52 +93,16 @@ data "aws_iam_policy_document" "lacework_audit_policy" {
 
   statement {
     sid = "APIGATEWAY"
-    actions = ["apigateway:GetApiKeys",
-      "apigateway:GetAuthorizers",
-      "apigateway:GetBasePathMappings",
-      "apigateway:GetClientCertificates",
-      "apigateway:GetDeployments",
-      "apigateway:GetDocumentationParts",
-      "apigateway:GetDocumentationVersions",
-      "apigateway:GetDomainNames",
-      "apigateway:GetGatewayResponses",
-      "apigateway:GetModels",
-      "apigateway:GetModelTemplate",
-      "apigateway:GetRequestValidators",
-      "apigateway:GetResources",
-      "apigateway:GetRestApis",
-      "apigateway:GetSdk",
-      "apigateway:GetSdkTypes",
-      "apigateway:GetStages",
-      "apigateway:GetTags",
-      "apigateway:GetUsagePlanKeys",
-      "apigateway:GetUsagePlans",
-    "apigateway:GetVpcLinks"]
-    resources = ["*"]
+    actions = ["apigateway:GET"]
+    resources = ["arn:aws:apigateway:*::/apikeys", "arn:aws:apigateway:*::/apikeys/*"]
```

Comment:
tested it with the change. both "*::/apikeys" and ":*::/apikeys/*" are needed

Reply:
> resources = ["*"]

Why not keep it consistent?

Reply:
Here is the conversation. Tasos mentioned about least priviledge rule. https://lacework.slack.com/archives/C01SW6S3NQY/p1702501578650959